### ISSUE TITLE
Champagne cork sign

### ISSUE THREAD

**Issue by azankl** [obophenotype/human-phenotype-ontology#7382]
**Preferred term label:**
Champagne cork sign

**Synonyms**
Champagne cork appearance
"Champagne cork" sign

**Definition (free text, please give PubMed ID)**
Characteristic appearance of the chest and abdomen in the sagittal view on prenatal ultrasound when the chest is small and the abdomen protruding. 
[PMID:31173381](https://pubmed.ncbi.nlm.nih.gov/31173381/) Figure 4E
[PMC:4166694](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4166694/) Figure 1C

**Parent term (use hpo.jax.org/app)**
Thoracic hypoplasia HP:0005257

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Thanatophoric dysplasia type I (https://www.omim.org/entry/187600)
Thanatophoric dysplasia type II (https://www.omim.org/entry/187601)

**Your nano-attribution (ORCID)**
Andreas Zankl 0000-0001-8612-1062

**Comment by pnrobinson** [comment]
added
